src/bezpath.rs

```diff
@@ -12,7 +12,7 @@ pub struct VelloBezPath {
 
 impl VelloBezPath {
     pub fn new() -> Self {
-        Self::default()
+        Self::default().with_trace(1.0)
```

Comment:
I was thinking the `Default` should be 1.0 instead of just inside the `fn new()`

```rust
impl Default for VelloBezPath {
    fn default() -> Self {
        Self {
            path: default(),
            trace: 1.0.
    }
}
```